docs/SCM-BestPractices/README.md

```diff
@@ -161,5 +163,5 @@ The following community members helped contribute to this guidance:
 * [Chris de Almeida, IBM](https://github.com/ctcpip)
 * [Christine Abernathy, F5 - project lead](https://github.com/caabernathy)
 * [Daniel Appelquist, Snyk - project lead](https://github.com/Torgo)
-* [Noam Dotan, Legit Security - project lead](https://github.com/)
+* [Noam Dotan, Legit Security - project lead](https://github.com/noamd-legit)
```

Comment:
this is fine but really should be in a separate PR

Reply:
Yeah, just didn't think it's worth it for this additional small fix